### PR TITLE
Add Sentry integration

### LINKED ISSUE #375: Sentry: Add source-map to sentry
For some reason sentry does not use the source map.
Checkout the sentry docs to add these.

https://sentry.io/all3dp-gmbh/printing-engine-client/issues/397339935/

### PR DESCRIPTION
**Pull request by jhnns**
This PR adds the Sentry integration for full source map support. Fixes #375.

With this Sentry integration, we should get:

- Source map support
- Commit integration for Sentry. For instance, by adding `Fixes SENTRY-317` to the commit message, Sentry is able to show which issues have been fixed by which release (and automatically close them).

See also https://docs.sentry.io/learn/cli/releases/

**Before this PR can be merged, we need to:**

- [x] Connect our Sentry project with the GitHub repository (I'm not allowed to do that)
- [x] Add the SENTRY_AUTH_TOKEN env variable to Travis

# Definition of Done

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [ ] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [ ] The PR has been reviewed
- [ ] There are no 'hacks' or shortcuts refactoring is prefered

## Tests
- [ ] Reasonably tested with high code coverage (unit/integration)

##	Testability
- [ ] All exported functions are testable
- [ ] There is no code execution in the global scope
- [ ] Default exports are avoided
- [ ] The code is concise, expressive and readable
- [ ] The code is functional and doesn’t use class syntax

## React components are
- [ ] Stateless and use pure rendering functions
- [ ] Are documented and tested in storybook
- [ ] Typed-checked
- [ ] The Sass is stickily using the BEM-Convention
- [ ] For each react component exists exactly one Sass-File
- [ ] The is no container styled
- [ ] There are no console errors or unwanted console logs

### CODE REVIEW

File: bin/purge-cloudflare-cache.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```
Comment (dreampulse) [inline bin/purge-cloudflare-cache.sh]: was macht `-eu`?

**Comment by dreampulse** [comment]
@jhnns lass uns auch das morgen zusammen machen

**Comment by dreampulse** [comment]
Die zwei offen Punte wurden von mir erledigt

**Comment by jhnns** [comment]
This PR also enables Sentry releases @Dansken:

bildschirmfoto 2017-11-15 um 15 23 54
---
bildschirmfoto 2017-11-15 um 15 25 54
---
bildschirmfoto 2017-11-15 um 15 26 26

**Pull request merged**